BioImage-Archive/bia-integrator
# Add reverse link indexes

For https://app.clickup.com/t/8695e0ecp with a max query time config variable in an older ticket should make sure we either get fast indexed responses or are aware we have an unindexed query (by the request failing)

Had to add another env var, should probably prioritise https://app.clickup.com/t/8695q6qjd soon

To test:
* Set `DB_INDEX_REFRESH=True` in `.env_compose`
* Start the api `docker compose up -d --build`
* Check `Indexes` for the bia_integrator collection, there should be lots of indexes named similar to `imagerepresentation_original_file_reference_uuid`  (one per `*_uuid` link, like the "reverse links" in the api)
